driver/sources/src/main/scala/org/virtuslab/ideprobe/ClassFixtures.scala

```diff
@@ -8,12 +8,15 @@ import org.virtuslab.ideprobe.reporting.AfterTestChecks
 
 trait RunningIntelliJPerSuiteBase {
 
-  private var workspace: Path = _
-  private var installed: InstalledIntelliJ = _
-  private var running: RunningIde = _
-  private var runningIntelliJFixture: RunningIntelliJFixture = _
-
-  final def intelliJ: RunningIntelliJFixture = runningIntelliJFixture
+  private var workspace: Option[Path] = None
+  private var installed: Option[InstalledIntelliJ] = None
+  private var running: Option[RunningIde] = None
+  private var runningIntelliJFixture: Option[RunningIntelliJFixture] = None
+
+  final def intelliJ: RunningIntelliJFixture = runningIntelliJFixture match {
```

Comment:
`runningIntelliJFixture.getOrElse(error("IntelliJ fixture not initialized"))`. 
`getOrElse` for more idiomatic handling options. 
As for `java.lang.Error` it signifies some more serious causes, like OutOfMemoryError, LinkageError, NoSuchMethodError, AssertionError etc. We just want a RuntimeException. Inside ide-probe code base, I think everywhere we use the `org.virtuslab.ideprobe.error` function, that just does `throw new RunimeException(msg)`, but is shorter to write, and whenever we need to introduce some special type of exception, we will be able to change it in one place.